modin/pandas/__init__.py

```diff
@@ -106,16 +106,19 @@ def _update_engine(publisher: Parameter):
     from modin.config import Backend, CpuCount
 
     if publisher.get() == "Ray":
-        from modin.engines.ray.utils import initialize_ray
+        if _is_first_update.get("Ray", True):
+            from modin.engines.ray.utils import initialize_ray
```

Comment:
what's the point of this change?

Reply:
Mostly for unification of the code in the if statements and dropping not necessary calls. Should probably be propagated to if statement with Native engine.